aries-site/src/pages/templates/content-layouts.mdx

```diff
@@ -3,17 +3,55 @@ import { Grid, ResponsiveContext } from 'grommet';
 import { CardGrid, ContentCard } from '../../components';
 import { DiagramPreview, Example } from '../../layouts';
 import { contentLayouts } from '../../data/structures';
-import { PageMarginAnatomy, ResponsiveContentLayoutExample } from '../../examples';
+import { ContainerDrivenLayout, NestedGrid, ResponsiveContentLayoutExample } from '../../examples';
 
-## Designing for Responsiveness
+Content layouts may be applied across different levels of information hierarchy.
+At a page template level, it can help with the layout of page sections. At an "organism" 
+level, it can coordinate how a title, piece of content, and related actions should be 
+laid out in relation to each other.
 
-App layouts should be adaptable to different screen sizes depending on the user input. Our goal as designers and developers at HPE should be to create optimal and accessible user experiences regardless of screen size or resolution.
+Understanding [Grid](/components/grid) is the key to creating successful content layouts.
+
+Looking for a quick start? [Jump to layout templates](#templates).
+
+## Responsive Design
+
+Layouts should be adaptable to different screen sizes depending on the user input. 
```

Comment:
"Depending on the user input" is a little confusing to me. Could we just simplify to:
```suggestion
Layouts should be adaptable to different screen sizes. 
```
or, rephrase?

Reply:
done